rqt_py_common/CMakeLists.txt

```diff
@@ -2,9 +2,24 @@ cmake_minimum_required(VERSION 2.8.3)
 project(rqt_py_common)
 # Load catkin and all dependencies required for this package
 find_package(catkin REQUIRED)
-catkin_package()
 catkin_python_setup()
 
+if(CATKIN_ENABLE_TESTING)
+  find_package(genmsg)
+  find_package(std_msgs)
+  add_message_files(
```

Comment:
Since the generated files should not be installed you should pass the `NOINSTALL` option (https://github.com/ros/genmsg/blob/cbb2112b302108f4c3e16f88baf78161e10be489/cmake/genmsg-extras.cmake.em#L77).